capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -107,8 +116,8 @@ function createAutocompleteListElement(collegeName, value) {
 }
 
 /**
- * Adds valid suggestions to list of stored suggestions
- * @param {string} validSuggestion
+ * Keeps track of chosen colleges
+ * @param {Array<string, collegeId>} validSuggestion
  */
 function keepTrackOfChosenColleges(validSuggestion) {
```

Comment:
Can you add a simple test verifying that this method works? No need to test removes, just that this method is roughly WAI...

Reply:
will do! give me five minutes @ccondit !

Reply:
haha, well that wasn't five minutes, but done!